stl/inc/memory

```diff
@@ -76,8 +76,10 @@ _NoThrowFwdIt uninitialized_copy_n(const _InIt _First, const _Diff _Count_raw, _
     if (0 < _Count) {
         auto _UFirst = _Get_unwrapped_n(_First, _Count);
         auto _UDest  = _Get_unwrapped_n(_Dest, _Count);
-        _Seek_wrapped(_Dest, _Uninitialized_copy_n_unchecked2(_UFirst, _Count, _UDest,
-                                 bool_constant<_Ptr_copy_cat<decltype(_UFirst), decltype(_UDest)>::_Really_trivial>{}));
+        _Seek_wrapped(_Dest,
+            _Uninitialized_copy_n_unchecked2(_UFirst, _Count, _UDest,
+                bool_constant<
+                    _Memcopy_is_safe<decltype(_UFirst), decltype(_UDest), _Memcopy_cat::_Copy_uninitialized>>{}));

```

Comment:
Style: The wrapping is becoming horrible, so this `bool_constant` should be extracted with `using`. (We've often said `using _Tag = BLAH` for this purpose.)